test/all.test.js

```diff
@@ -86,9 +86,34 @@ function tests() {
       expect(this.folder.associations.ancestors.through.model.attributes).to.have.property('testThroughForeignKey');
       expect(this.folder.associations.ancestors.foreignKey).to.equal('testThroughKey');
       expect(this.folder.associations.children.foreignKey).to.equal('testForeignKey');
+			expect(this.folder.associations.ancestors.through.model.tableName).to.equal('foldersancestors');
```

Comment:
I'm not sure why this line is indented improperly, it showed up fine for me in my editor (Atom)